pkg/envtest/server.go

```diff
@@ -156,10 +156,11 @@ func (te *Environment) Stop() error {
 }
 
 // getAPIServerFlags returns flags to be used with the Kubernetes API server.
+// it returns empty slice for api server defined defaults to be applied if no args specified
 func (te Environment) getAPIServerFlags() []string {
 	// Set default API server flags if not set.
 	if len(te.KubeAPIServerFlags) == 0 {
-		return DefaultKubeAPIServerFlags
+		return []string{}
```

Comment:
should we alias the above global variable too?

Reply:
Isn't the `DefaultKubeAPIServerFlags` unused after this change?

Reply:
I forgot to remove it 🤦‍♀ thanks